spec/presenters/claim/transfer_claim_presenter_spec.rb

```diff
@@ -102,80 +105,72 @@
     end
   end
   
-  describe 'copied from spec/presenters/transfer_claim_presenter_spec.rb' do
-    let(:claim) { instance_double(Claim::TransferClaim, transfer_detail: double) }
-    let(:presenter) { Claim::TransferClaimPresenter.new(claim, view) }
-  
-    it { expect(presenter).to be_instance_of(Claim::TransferClaimPresenter) }
-    it { expect(presenter).to be_kind_of(Claim::BaseClaimPresenter) }
-  
-    it 'has disbursements' do
-      expect(presenter.can_have_disbursements?).to eq(true)
-    end
-  
-    let(:case_conclusions) do
-      {
-        '10' => 'Trial',
-        '20' => 'Retrial',
-        '30' => 'Cracked',
-        '40' => 'Cracked before retrial',
-        '50' => 'Guilty plea'
-      }
+  it 'has disbursements' do
+    expect(presenter.can_have_disbursements?).to eq(true)
```

Comment:
```suggestion
    expect(presenter.can_have_disbursements?).to be_truthy
```

Reply:
Rubocop wanted this to be

```
it { expect(presenter).to be_can_have_disbursements }
```

so I re-write it as

```ruby
  describe '#can_have_disbursements?' do
    subject { presenter.can_have_disbursements? }

    it { is_expected.to be_truthy }
  end
```

It might do to rename the method.